tests/foreman/smoke/test_cli_smoke.py

```diff
@@ -729,5 +729,6 @@ def test_end_to_end(self):
                 .format(result.stderr, result.return_code)
             )
             # Verify if package is installed by query it
-            result = vm.run('rpm -q {0}'.format(package_name))
+            result = vm.run('rpm -qa | grep {0}'.format(package_name))
+            self.assertTrue(result.stdout)
```

Comment:
`assertTrue` have more meaning when you are asserting boolean objects, in this case what you mean should be `assertGreater(len(result.stdout), 0)`. With the latter someone else reading the code will figure out that you expect some output and don't need to dig into the code in order to check if that stdout instance is a boolean one or not.